packages/google_mlkit_document_scanner/README.md

```diff
@@ -0,0 +1,104 @@
+- Gallery import 
+- Editing functionalities
+
+**NOTE** Since [Google's Document Scanner](https://developers.google.com/ml-kit/vision/doc-scanner) API is still in Beta and only supports Android. Stay tune for updates in their website.
+
+**PLEASE READ THIS** before continuing or posting a [new issue](https://github.com/flutter-ml/google_ml_kit_flutter/issues):
+
+- [Google's ML Kit](https://developers.google.com/ml-kit) was build only for mobile platforms: iOS and Android apps.
+
+- This plugin is not sponsored or maintained by Google. The [authors](https://github.com/flutter-ml/google_ml_kit_flutter/blob/master/AUTHORS) are developers excited about Machine Learning that wanted to expose Google's native APIs to Flutter.
+
+- Google's ML Kit APIs are only developed natively for iOS and Android. This plugin uses Flutter Platform Channels as explained [here](https://docs.flutter.dev/development/platform-integration/platform-channels).
+
+  Messages are passed between the client (the app/plugin) and host (platform) using platform channels as illustrated in this diagram:
+
+  <p align="center" width="100%">
+    <img src="https://docs.flutter.dev/assets/images/docs/PlatformChannels.png"> 
+  </p>
+
+  Messages and responses are passed asynchronously, to ensure the user interface remains responsive. To read more about platform channels go [here](https://docs.flutter.dev/development/platform-integration/platform-channels).
+
+  Because this plugin uses platform channels, no Machine Learning processing is done in Flutter/Dart, all the calls are passed to the native platform using `MethodChannel` in Android and `FlutterMethodChannel` in iOS, and executed using Google's native APIs. Think of this plugin as a bridge between your app and Google's native ML Kit APIs. This plugin only passes the call to the native API and the processing is done by Google's API. It is important that you understand this concept when it comes to debugging errors for your ML model and/or app.
+
+- Since the plugin uses platform channels, you may encounter issues with the native API. Before submitting a new issue, identify the source of the issue. You can run both iOS and/or Android native [example apps by Google](https://github.com/googlesamples/mlkit) and make sure that the issue is not reproducible with their native examples. If you can reproduce the issue in their apps then report the issue to Google. The [authors](https://github.com/flutter-ml/google_ml_kit_flutter/blob/master/AUTHORS) do not have access to the source code of their native APIs, so you need to report the issue to them. If you find that their example apps are okay and still you have an issue using this plugin, then look at our [closed and open issues](https://github.com/flutter-ml/google_ml_kit_flutter/issues). If you cannot find anything that can help you then report the issue and provide enough details. Be patient, someone from the community will eventually help you.
+
+## Requirements
+
```

Comment:
Add section for iOS and mentioned that it is not supported yet.

Reply:
Yes, i added not support for iOS, but i am updating the requirement for iOS as you mentioned.

Reply:
What I mean is: adding:

```
### iOS

Not supported
```

Remove what you added and replace with above.